flake.nix

```diff
@@ -34,10 +34,6 @@
       url = "https://github.com/neonphog.keys";
       flake = false;
     };
-    keys_thedavidmeister = {
```

Comment:
why remove this key? @thedavidmeister is actively working with us

Reply:
indeed

Reply:
I just removed this because the keys from github changed and I wasn't sure if this is supposed to be.

Reply:
the keys changed? as in at my end?

Reply:
Your keys on github are not the same as a few weeks ago. That's all I know.

Reply:
oh, i added a new one for a new device yeah

Reply:
Added back the keys and updated the lock file